frontend/.env.example

```diff
@@ -1,4 +0,0 @@
-# Example environment variables for frontend. Any "<example>"s should be
-# replaced with the real values.
-NEXT_PUBLIC_BACKEND_URL="http://localhost:3001"
-NEXT_PUBLIC_FIREBASE_SETTINGS='<example>'

```

Comment:
Same as for `backend/.env.example`, can we restore this unless there was a reason you deleted it?